app/graphql/mutations/profile/associate_rules.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Profile
+    # Mutation to associate systems with a profile
+    class AssociateRules < BaseMutation
+      graphql_name 'associateRules'
+
+      argument :id, ID, required: true
+      argument :rule_ids, [ID], required: true
```

Comment:
@bastilian Would frontend send UUIDs or does it use ref_ids instead?

Reply:
@vkrizan @skateman The Frontend does send ref_ids at the moment. I'm not opposed to changing that at all btw., but maybe later on, or if we encounter issues with using the ref_id.

Reply:
@skateman @bastilian What do you guys suggest then? Can we use UUID ids to keep the APIs consistent and update the frontend to do the use them?

Reply:
I recommend using UUIDs like the REST API if using `rule_ids`. If we want to support ref_ids in addition, we should add another argument `rule_ref_ids`.